modules/github-bots/sdk/github.go

```diff
@@ -579,6 +579,22 @@ func (c GitHubClient) GetFileContent(ctx context.Context, owner, repo, path, ref
 	return content, nil
 }
 
+// SearchFilenameInRepository searches for a filename in a specific repository
```

Comment:
Looks like this file doesn't have tests at all, sure would be nice to add them at some point. Not your doing and not asking you to add them, just jotting it down.

Reply:
I've added an integration test as per your request. It is not the best but it gives a playground for testing.